src/views/ProjectView.vue

```diff
@@ -1,9 +1,127 @@
 <script setup>
-    import ToggleButton from '../components/ToggleButton.vue'
-    import ImportButton from '../components/ImportButton.vue'
+import { ref } from 'vue'
+import { useRouter } from 'vue-router'
+import { useStore } from 'vuex'
+import { fetchApiCall } from '../utils/api'
+import ToggleButton from '../components/ToggleButton.vue'
+
+const router = useRouter()
+const store = useStore()
+const isPopupVisible = ref(false)
+const uniqueDataSessions = ref([])
+const newSessionName = ref('')
+const errorMessage = ref('')
+
+const getDataSessions = async () => {
+  try {
```

Comment:
Do we need all of this in a try/except block? It's good practice to limit the scope of a try block to just the line or lines of code that may generate an exception - in this case, probably just the fetchApiCall() method.